site/docs/build-ref.html

```diff
@@ -291,7 +291,7 @@ <h3 id="lexi">Lexical specification of a label</h3>
   The precise details of allowed target names are below.
 </p>
 
-<h4 id="name">Target names, <code>//...:<b>target-name</b></code></h4>
+<h4 id="name">Target names, <code>//xxx:<b>target-name</b></code></h4>
```

Comment:
It took me some time to understand what's being communicated . But now I get it. This section says "ignore the package name, focus on target-name". The next section says "ignore the target name, focus on the package name". 

I agree with your basic point. I worry that an explicit string might blur the purpose, i.e. a reader might incorrectly think its naming is significant. But I can't think of anything better, and I agree what's there now is lacking.

Sensitivity suggestion: how about `xyz`, since `xxx` can have unfavorable associations?

Reply:
Fair enough. Switched to `xyz`.